src/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  margin: 0;
+}
+
+div {
```

Comment:
- Don't use tags for styling elements
- div are block by default so this rule is redundant